conformance/lang/values-types-and-variables/boolean-literal.balt

```diff
@@ -0,0 +1,161 @@
+Test-Case: output
+Description: Test boolean type descriptor.
```

Comment:
Test is for the different boolean literals against the boolean type-descriptor?

Reply:
Yes. will update the Description.